src/Breadcrumb/BreadcrumbStep.tsx

```diff
@@ -0,0 +1,35 @@
+/* eslint react/prop-types: 0 */
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+
+import { BreadcrumbStep as _BreadcrumbStep, BreadcrumbLink as _BreadcrumbLink } from './styled';
+
+export type LocalBreadcrumbStepProps = {
+  color?: string;
+  href: string;
+  isCurrent?: boolean;
+  text: string;
```

Comment:
I think we should make use of `children` instead of `text`. Check out [this article](https://sid.studio/just-use-children/) on reasons why children may make more sense.